examples/full-example/.mendelrc

```diff
@@ -53,23 +53,19 @@ transforms: # a list of all available transforms for all envs and types
         - istanbul
 
 types:
-  json:
+  json: # Type id. Doesn't have to be file extension.
     extensions:
       - .json
-    parser: json-parse
-    parser-to-type: js
-    include-node_modules: true
+    parser: json-parse # Uses parser defined in transforms.
+    parser-to-type: js # Maps to type defined below
+    include-node_modules: true # Include files matching node_modules/* path.
   js:
-    outlet:
-      plugin: mendel-bundle-browser-pack
```

Comment:
the implementation is not using this at all. Frankly, I am still contemplating about this but since we are not honoring this config, I thought it would be a good idea before people start to copy it.

See "bundles" having the outlet property. We can talk about this offline. I need some guidance.